akka-actor/src/main/scala/akka/dispatch/AbstractDispatcher.scala

```diff
@@ -327,6 +329,8 @@ abstract class MessageDispatcherConfigurator(_config: Config, val prerequisites:
     def configurator(executor: String): ExecutorServiceConfigurator = executor match {
       case null | "" | "fork-join-executor" ⇒ new ForkJoinExecutorConfigurator(config.getConfig("fork-join-executor"), prerequisites)
       case "thread-pool-executor"           ⇒ new ThreadPoolExecutorConfigurator(config.getConfig("thread-pool-executor"), prerequisites)
+      case "affinity-pool-executor"         ⇒ new AffinityPoolConfigurator(config.getConfig("affinity-pool-executor"), prerequisites)
+
       case fqcn ⇒
```

Comment:
If we make the affinity-pool a separate module "akka-affinity-pool", then we can instead provide the fqcn of the AffinityPoolConfigurator and have it loaded like that instead of "affinity-pool-executor". Also makes it easier to collaborate on the affinity pool (if it is a separate module)